sphinx/source/developers/logging_cpp.rst

```diff
@@ -57,17 +57,29 @@ The logging app defines :cpp:class:`ccfapp::LoggerHandlers`, which creates and i
     :end-before: SNIPPET_END: get
```

Comment:
I wonder if it's worth adding a new page/section that describes the different types and options on handlers without mentioning the logging app specifically? I think we'll need one sooner or later to describe the different types/flags on handlers. We could always link to the logging app as examples (with back links too). Thinking about it, we should doxygen `handlerregistry.h` and get most of this for free. What do you think?

Reply:
I agree! This is where we should aim, ideally with most of this coming 'for free' from doxygen. I think the next step is to split the thing apps see from the thing the framework sees (ie - they get an opaque `Handle` with the methods, we see the raw struct state), then the app side gets doxygen'd.

Naming is awkward - what's a 'handle' for a 'handler'? Maybe we should go more HTTP-heavy and talk about installing _resources_ or _endpoints_ rather than RPC handlers?

Anyway I think this is all future work.

Reply:
doxygen + links is absolutely the way to go! For naming, I think we should pick a popular framework, align, and reference explicitly.